gsms.py

```diff
@@ -153,6 +157,176 @@ def get_win_path(folder_id: str) -> str:
     # Free memory used by pointer
     _CoTaskMemFree(path_pointer)
 
+    print(path)
+
+    return path
+
+
+def copy_image(src_image: str, dst_image: str) -> None:
+    """
+    Copies an image from src_image to dst_image if the dst is empty or different
+
+    Parameters
+    ----------
+    src_image: str
+        Source path of the image
+    dst_image: str
+        Destination path of the image
+
+    Returns
+    -------
+    None
```

Comment:
Doesn't return anything so section is not necessary.

Reply:
Will remove